theano/tensor/nnet/corr.py

```diff
@@ -274,6 +281,7 @@ def c_code_helper(self, bottom, weights, top, sub, height=None, width=None):
     int dilW = %(params)s->dilW;
     int padH = %(params)s->padH;
     int padW = %(params)s->padW;
+    int numgroups = %(params)s->num_groups;
```

Comment:
As this file have c code in the Op method and not in separate c file, you need to bump the version number returned by the method c_code_cache_version() of each op you changed the c code.

Op that have its c code in separate file, don't implement that method, so you are fine.